### ISSUE TITLE
Can't setup email sending

### ISSUE THREAD

**Issue by Turbito** [tootsuite/mastodon#2312]
Hello,
I had try to configure the emailing setup for the registration of my instance but I can't get it

My instance is piscolabis.me
```
SMTP_SERVER=smtp.gmail.com
SMTP_PORT=587
SMTP_LOGIN=my google username@gmail.com
SMTP_PASSWORD= my gmail password
SMTP_FROM_ADDRESS=my google username@gmail.com
#SMTP_DOMAIN= # defaults to LOCAL_DOMAIN
#SMTP_DELIVERY_METHOD=smtp # delivery method can also be sendmail
#SMTP_AUTH_METHOD=plain
#SMTP_OPENSSL_VERIFY_MODE=peer
#SMTP_ENABLE_STARTTLS_AUTO=true
```

I have enabled "less secure apps" for google too

Also tried with sparkpost with no result

```
SMTP_SERVER=smtp.sparkpostmail.com
SMTP_PORT=587
SMTP_LOGIN=SMTP_Injection
SMTP_PASSWORD=myapikey
SMTP_FROM_ADDRESS=piscolabis.me
SMTP_AUTH_METHOD=AUTH LOGIN
```

I have also tried the solutions proposed for two similars issues opened here

Regards.

**Comment by ashfurrow** [comment]
Hmm, interesting. Can you check your server logs to see what kind of error messages you're getting? (I had to set `SMTP_OPENSSL_VERIFY_MODE` to `none` to get mine to work with Mailgun.)

**Comment by ashfurrow** [comment]
I think the worker logs would be most relevant; look for anything related to SMTP failures. _Where_ the logs are depends on how you're hosting the instance. Are you using Docker or Vagrant? Neither?

**Comment by Turbito** [comment]
Neither, I followed the production guide  (https://github.com/tootsuite/documentation/blob/master/Running-Mastodon/Production-guide.md)

**Comment by ashfurrow** [comment]
I see, I'm afraid I'm not sure where the logs are (I use Docker). I think @wxcafe hosts outside Docker, maybe they can help point us in the right direction.

**Comment by wxcafe** [comment]
you can get logs by using `journalctl -u mastodon-web`. Look for anything related to smtp and post it here. You don't need to enable the less secure apps from gmail afaict, but maybe changing the `SMTP_DOMAIN` setting would help

**Comment by Turbito** [comment]
this is what it generates when I ask for a new confirmation email for a test user
```
Apr 22 23:53:46 piscolabis bundle[1106]: [85279657-a902-40db-b309-40d34d329771] method=GET path=/auth/sign_in format=html controller=Auth::SessionsController action=new status=200 duration=53.83 view=48.83 db=0.00
Apr 22 23:53:49 piscolabis bundle[1106]: [1b6e6d46-7dbc-4f05-9366-840eade296c1] method=GET path=/auth/confirmation/new format=html controller=Auth::ConfirmationsController action=new status=200 duration=19.51 view=13.86 db=0.00
Apr 22 23:53:52 piscolabis bundle[1106]: [93082733-52f7-4256-ae8d-bcd0ff538620] [ActiveJob] Enqueued ActionMailer::DeliveryJob (Job ID: d28af711-e227-421a-b89a-f51e237204e0) to Sidekiq(mailers) with arguments: "UserMailer", "confirmation_instructions", "deliver_now", #<GlobalID:0x007f91f497e428 @uri=#<URI::GID gid://mastodon/User/2>>, "EL2aqQ752uCe1nmxEp6D", {}
Apr 22 23:53:52 piscolabis bundle[1106]: [93082733-52f7-4256-ae8d-bcd0ff538620] method=POST path=/auth/confirmation format=html controller=Auth::ConfirmationsController action=create status=302 duration=63.41 view=0.00 db=1.51 location=https://piscolabis.me/auth/sign_in
Apr 22 23:53:52 piscolabis bundle[1106]: [3432fff4-247a-456c-b115-fcd375e9507e] method=GET path=/auth/sign_in format=html controller=Auth::SessionsController action=new status=200 duration=7.13 view=5.15 db=0.00
Apr 22 23:54:39 piscolabis bundle[1106]: [6c59199a-de21-45fd-ac1d-d1e1ebe6a98d] method=GET path=/api/v1/timelines/public format=*/* controller=Api::V1::TimelinesController action=public status=200 duration=398.41 view=331.57 db=5.60
Apr 22 23:54:40 piscolabis bundle[1106]: [02a6e662-0c29-41df-8d99-157353b81442] method=GET path=/about/more format=html controller=AboutController action=more status=200 duration=7.32 view=5.49 db=0.00
Apr 22 23:54:40 piscolabis bundle[1106]: [c16e6514-e950-43a0-ba0e-440cdd0422b2] method=GET path=/auth/sign_up format=html controller=Auth::RegistrationsController action=new status=302 duration=2.48 view=0.00 db=0.00 location=https://piscolabis.me/
```

**Comment by wxcafe** [comment]
that means it's been sent to sidekiq. Can you go to piscolabis.me/sidekiq and see if there are a lot of enqueued items? Please also post your sidekiq log (`journalctl -u mastodon-sidekiq`)

**Comment by Turbito** [comment]
Yes, there is a lot of retries queued

ok, that is generated for the confirmation email
```Apr 23 00:09:45 piscolabis bundle[5750]: 2017-04-22T22:09:45.501Z 5750 TID-or2xkg28s ActionMailer::DeliveryJob JID-36abcf0340084b68d9c5a4a1 INFO: start
Apr 23 00:09:45 piscolabis bundle[5750]: 2017-04-22T22:09:45.509Z 5750 TID-or2xkg28s ActionMailer::DeliveryJob JID-36abcf0340084b68d9c5a4a1 INFO: fail: 0.008 sec
Apr 23 00:09:45 piscolabis bundle[5750]: 2017-04-22T22:09:45.510Z 5750 TID-or2xkg28s WARN: {"context":"Job raised exception","job":{"class":"ActiveJob::QueueAdapters::SidekiqAdapter::JobWrapper","wrapped":"ActionMailer::DeliveryJob","queue":"mailers","args":[{"job_class":"ActionMailer::DeliveryJob","job_id":"0d689f23-6ec2-4723-a0ba-c47d5aa4cbfa","queue_name":"mailers","priority":null,"arguments":["UserMailer","confirmation_instructions","deliver_now",{"_aj_globalid":"gid://mastodon/User/2"},"EL2aqQ752uCe1nmxEp6D",{"_aj_symbol_keys":[]}],"locale":"es"}],"retry":true,"jid":"36abcf0340084b68d9c5a4a1","created_at":1492898369.0159645,"enqueued_at":1492898985.5007613,"error_message":"Error while trying to deserialize arguments: Couldn't find User with 'id'=2","error_class":"ActiveJob::DeserializationError","failed_at":1492898369.547119,"retry_count":5,"retried_at":1492898985.5103714},"jobstr":"{\"class\":\"ActiveJob::QueueAdapters::SidekiqAdapter::JobWrapper\",\"wrapped\":\"ActionMailer::DeliveryJob\",\"queue\":\"mailers\",\"args\":[{\"job_class\":\"ActionMailer::DeliveryJob\",\"job_id\":\"0d689f23-6ec2-4723-a0ba-c47d5aa4cbfa\",\"queue_name\":\"mailers\",\"priority\":null,\"arguments\":[\"UserMailer\",\"confirmation_instructions\",\"deliver_now\",{\"_aj_globalid\":\"gid://mastodon/User/2\"},\"EL2aqQ752uCe1nmxEp6D\",{\"_aj_symbol_keys\":[]}],\"locale\":\"es\"}],\"retry\":true,\"jid\":\"36abcf0340084b68d9c5a4a1\",\"created_at\":1492898369.0159645,\"enqueued_at\":1492898985.5007613,\"error_message\":\"Error while trying to deserialize arguments: Couldn't find User with 'id'=2\",\"error_class\":\"ActiveJob::DeserializationError\",\"failed_at\":1492898369.547119,\"retry_count\":4,\"retried_at\":1492898605.6431997}"}
Apr 23 00:09:45 piscolabis bundle[5750]: 2017-04-22T22:09:45.512Z 5750 TID-or2xkg28s WARN: ActiveJob::DeserializationError: Error while trying to deserialize arguments: Couldn't find User with 'id'=2
Apr 23 00:09:45 piscolabis bundle[5750]: 2017-04-22T22:09:45.512Z 5750 TID-or2xkg28s WARN: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activerecord-5.0.2/lib/active_record/relation/finder_methods.rb:353:in `raise_record_not_found_exception!'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activerecord-5.0.2/lib/active_record/relation/finder_methods.rb:479:in `find_one'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activerecord-5.0.2/lib/active_record/relation/finder_methods.rb:458:in `find_with_ids'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activerecord-5.0.2/lib/active_record/relation/finder_methods.rb:66:in `find'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activerecord-5.0.2/lib/active_record/querying.rb:3:in `find'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activerecord-5.0.2/lib/active_record/core.rb:151:in `find'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/globalid-0.3.7/lib/global_id/locator.rb:130:in `locate'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/globalid-0.3.7/lib/global_id/locator.rb:155:in `block in locate'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/globalid-0.3.7/lib/global_id/locator.rb:165:in `block in unscoped'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activerecord-5.0.2/lib/active_record/scoping/default.rb:35:in `block in unscoped'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activerecord-5.0.2/lib/active_record/relation.rb:351:in `scoping'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activerecord-5.0.2/lib/active_record/scoping/default.rb:35:in `unscoped'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/globalid-0.3.7/lib/global_id/locator.rb:165:in `unscoped'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/globalid-0.3.7/lib/global_id/locator.rb:155:in `locate'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/globalid-0.3.7/lib/global_id/locator.rb:17:in `locate'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/arguments.rb:111:in `deserialize_global_id'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/arguments.rb:97:in `deserialize_argument'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/arguments.rb:51:in `block in deserialize'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/arguments.rb:51:in `map'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/arguments.rb:51:in `deserialize'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/core.rb:128:in `deserialize_arguments'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/core.rb:118:in `deserialize_arguments_if_needed'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/execution.rb:32:in `perform_now'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/execution.rb:22:in `block in execute'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:126:in `call'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:506:in `block (2 levels) in compile'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:455:in `call'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:448:in `block (2 levels) in around'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:286:in `block (2 levels) in halting'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/railtie.rb:26:in `block (4 levels) in <class:Railtie>'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/execution_wrapper.rb:85:in `wrap'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/reloader.rb:68:in `block in wrap'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/execution_wrapper.rb:81:in `wrap'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/reloader.rb:67:in `wrap'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/railtie.rb:25:in `block (3 levels) in <class:Railtie>'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:391:in `instance_exec'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:391:in `block in make_lambda'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:285:in `block in halting'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:447:in `block in around'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:455:in `call'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:101:in `__run_callbacks__'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:750:in `_run_execute_callbacks'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/callbacks.rb:90:in `run_callbacks'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/execution.rb:20:in `execute'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activejob-5.0.2/lib/active_job/queue_adapters/sidekiq_adapter.rb:40:in `perform'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/processor.rb:167:in `execute_job'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/processor.rb:139:in `block (5 levels) in process'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/rails.rb:90:in `block in call'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/activesupport-5.0.2/lib/active_support/execution_wrapper.rb:85:in `wrap'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/rails.rb:89:in `call'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/processor.rb:135:in `block (4 levels) in process'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/middleware/chain.rb:128:in `block in invoke'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-unique-jobs-5.0.0/lib/sidekiq_unique_jobs/server/middleware.rb:15:in `call'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/middleware/chain.rb:130:in `block in invoke'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/middleware/server/logging.rb:10:in `call'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/middleware/chain.rb:130:in `block in invoke'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/middleware/server/retry_jobs.rb:74:in `call'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/middleware/chain.rb:130:in `block in invoke'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/middleware/chain.rb:133:in `invoke'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/processor.rb:134:in `block (3 levels) in process'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/logging.rb:32:in `with_context'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/processor.rb:132:in `block (2 levels) in process'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/processor.rb:183:in `stats'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/processor.rb:131:in `block in process'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq.rb:35:in `block in <module:Sidekiq>'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/processor.rb:126:in `process'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/processor.rb:82:in `process_one'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/processor.rb:70:in `run'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/util.rb:17:in `watchdog'
Apr 23 00:09:45 piscolabis bundle[5750]: /home/mastodon/live/vendor/bundle/ruby/2.4.0/gems/sidekiq-4.2.10/lib/sidekiq/util.rb:26:in `block in safe_thread'
Apr 23 00:09:47 piscolabis bundle[5750]: 2017-04-22T22:09:47.089Z 5750 TID-or2x8ckig ProcessingWorker JID-5feebf796cfaf3417d403d3c INFO: start
Apr 23 00:09:47 piscolabis bundle[5750]: 2017-04-22T22:09:47.095Z 5750 TID-or2xawf0c RemoteProfileUpdateWorker JID-32648ae12085db71c51dba83 INFO: start
Apr 23 00:09:47 piscolabis bundle[5750]: 2017-04-22T22:09:47.101Z 5750 TID-or2x8ckig ProcessingWorker JID-5feebf796cfaf3417d403d3c INFO: done: 0.012 sec
Apr 23 00:09:47 piscolabis bundle[5750]: 2017-04-22T22:09:47.107Z 5750 TID-or2xawf0c RemoteProfileUpdateWorker JID-32648ae12085db71c51dba83 INFO: done: 0.013 sec

```

**Comment by wxcafe** [comment]
Wow uh that's /weird/. It looks like ActiveMailer is trying to send this email with some account id but can't find it in the database? idk. Pinging @Gargron :<

**Comment by Turbito** [comment]
Ok, guys, I get it.
That error triggered me and looked in the users database.
For a weird reason, all no confirmed users were gone...
I just tried to recreate another acc and now works like a charm
Many thanks you

**Comment by wxcafe** [comment]
Well, no problems

**Comment by psydwannabe** [comment]
Leaving this here for anyone else who might, like me, think they had a related issue:

I saw this error for all email notifications after a server migration and assumed there was some kind of configuration or caching error causing the problem.

The issue was caused by a firewall with my hosting provider. When I tried to telnet to my mail provider, it timed out-- that was the key to finding the problem.  I host with Scaleway which blocks those ports by default.  Their FAQ has a line about this (and how to correct it) here: https://www.scaleway.com/faq/servers/network/#-I-cannot-send-any-email
